internal/provider/resource_ger_ruleset.go

```diff
@@ -0,0 +1,231 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/squadcast/terraform-provider-squadcast/internal/api"
+	"github.com/squadcast/terraform-provider-squadcast/internal/tf"
+)
+
+func resourceGERRuleset() *schema.Resource {
+	return &schema.Resource{
+		Description: "GER Ruleset is a collection of rules that are used to route alerts to the correct service based on the alert source.",
+
+		CreateContext: resourceGERRulesetCreate,
+		ReadContext:   resourceGERRulesetRead,
+		UpdateContext: resourceGERRulesetUpdate,
+		DeleteContext: resourceGERRulesetDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceGERRulesetImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Description: "GER Ruleset id.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"ger_id": {
+				Description: "GER id.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"alert_source": {
+				Description: "GER Ruleset alert source. Find all alert sources [here](https://www.squadcast.com/integrations)",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"alert_source_version": {
+				Description: "GER Ruleset alert source version.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"alert_source_shortname": {
+				Description: "GER Ruleset alert source shortname.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"catch_all_action": {
+				Description: "Catch all action allows user to select a service, this will act as a fallback service and alert source will be routed to the selected service.",
```

Comment:
rephrase this:
Catch-all action allows the user to select a service, this will act as a fallback service and an alert source will be routed to the selected service when none of the rules match.